ush/parsing_namelists_FV3.sh

```diff
@@ -55,11 +55,6 @@ cat > input.nml <<EOF
   chksum_debug = $chksum_debug
   dycore_only = $dycore_only
   ccpp_suite = $CCPP_SUITE
-  fdiag = $FDIAG
-  fhmax = $FHMAX
```

Comment:
If these are removed, where is the atmosphere output frequency set now?

Reply:
See: https://github.com/ufs-community/ufs-weather-model/commit/74a34d225fb76702e48c0a27534514992ac2ce77

Reply:
Okay, so I guess nfhout, etc. in model_configure are controlling the output frequency and the new output_fh is only needed if you want custom output times?

Also, looking at the associated UFS PR, it looks like there were also changes to restart_interval that now need to be made in the workflow that I don't see in this PR.

Reply:
I'm just trying to get things set up for P7 (not p7.1/aerosols etc) so this was a "make it run" PR particularly as I understand development is supposed to be kept to a minimum on this branch.  Do I need to go back and make this more all encompassing?

Reply:
IMO, aerosols need to continue working until the sprint branch can handle them. I can do the tricky stuff for restart_interval, if you'd like.